src/conduit-lwt-unix/conduit_lwt_unix_ssl.dummy.ml

```diff
@@ -16,10 +16,13 @@
  *)
 
 module Client = struct
+  type verify = { hostname : bool; ip : bool }
+
+  let default_verify = { hostname = true; ip = true }
   let default_ctx = `Ssl_not_available
   let create_ctx ?certfile:_ ?keyfile:_ ?password:_ () = default_ctx
 
-  let connect ?(ctx = default_ctx) ?src:_ ?hostname:_ _sa =
+  let connect ?(ctx = default_ctx) ?src:_ ?hostname:_ ~ip:_ ~verify:_ _sa =
```

Comment:
Please use ?ip and ?verify, looks like the CI may not be verifying this, but if the interface declares optional labeled arguments, the implementations cannot use non-optional ones, even if it ignores them.

Reply:
Thanks, I've changed it